static/css/tabzilla/tabzilla.css

```diff
@@ -0,0 +1,49 @@
+/* Taken from https://github.com/mozilla/tabzilla/blob/master/css/tabzilla.css.
+ * The changes made are highlighted throughout this file.
+ */
+#tabzilla {
+  position: relative;
+  float: right;
+}
+#tabzilla a {
+  position: relative;
+  display: block;
+  width: 147px;
+  height: 37px;
+  text-indent: 120%;
+  white-space: nowrap;
+  overflow: hidden;
+  /* Changed from '/media/img/tabzilla-static.png' to fit our static folders
+   * layout.
+   */
+  background-image: url('../../img/tabzilla/tabzilla-static.png');
+  background-repeat: no-repeat;
+  z-index: 2;
+}
+@media (-webkit-min-device-pixel-ratio: 1.5), (min--moz-device-pixel-ratio: 1.5), (-o-min-device-pixel-ratio: 3/2), (min-resolution: 1.5dppx) {
+  #tabzilla a {
+    /* Changed from '/media/img/tabzilla-static-high-res.png' to fit our static
+     * folders layout.
+     */
+    background-image: url("../../img/tabzilla/tabzilla-static-high-res.png");
+    -webkit-background-size: 147px 37px;
+    background-size: 147px 37px;
+  }
+}
+#tabzilla:before {
+  position: absolute;
+  display: block;
+  left: 28px;
+  top: 0;
+  width: 88px;
+  height: 26px;
+  content: '';
+  /* Changed from 'transparent' to be the same color we had on AMO, following
+   * what's explained here:
+   * https://github.com/mozilla/tabzilla/#changing-the-color-of-the-wordmark
+   * This color is the "Mozilla Charcoal" from
+   * https://www.mozilla.org/en-US/styleguide/identity/mozilla/color/
+   */
+  background-color: #4D4E53;
```

Comment:
After an email exchange with @schalkneethling, he advised me to pick this color from the official styleguide ;)